### ISSUE TITLE
Documentation:  Adding an RStudio style cheatsheet

### ISSUE THREAD

**Issue by zdz2101** [pharmaverse/admiral#1529]
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

Would be interesting to create an RStudio-style cheatsheet for admiral, especially popularly used functions/family of functions:

https://github.com/rstudio/cheatsheets/blob/main/README.md

**Comment by bms63** [comment]
Love it!  @pharmaverse/admiral

**Comment by bundfussr** [comment]
Yes, great idea. I would propose to do it for the 1.0.0 release because https://github.com/pharmaverse/admiral/issues/1522 will cause a lot of changes and after the clean up it should be easier to create the cheatsheet.

**Comment by rossfarrugia** [comment]
agree great idea for 1.0.0 release. @yangc71 created a great poster for admiral to share internally here recently which could be a starting point when the time comes for this one - it will need updating to go deeper into the deeper functional level to act as a cheat sheet but there might be some re-use.

**Comment by bms63** [comment]
Let's wait to implement until admiral is more stable.  Look into ways to automate with checking for warning/errors for deprecation but also in building the cheat sheet (i.e. no powerpoint!)

**Comment by rossfarrugia** [comment]
Love this new GH action for stale issues, very cool ❤️

**Comment by bms63** [comment]
I obey my bot-overlords.  We will get it sorted.
